include/seqan3/alignment/matrix/trace_directions.hpp

```diff
@@ -47,3 +47,26 @@ namespace seqan3
 template <>
 constexpr bool add_enum_bitwise_operators<seqan3::detail::trace_directions> = true;
 } // namespace seqan3
+
+#include <seqan3/io/stream/debug_stream.hpp>
+namespace seqan3
+{
+
+/*!\brief All trace_directions can be printed as ascii or as utf8 to the seqan3::debug_stream.
+ * \param s The seqan3::debug_stream.
+ * \param trace The trace direction.
+ * \relates seqan3::debug_stream_type
+ */
+inline debug_stream_type & operator<<(debug_stream_type & s, detail::trace_directions trace)
+{
+    static char const * unicode[8]{"↺","↖","↑","↖↑","←","↖←","↑←","↖↑←"};
```

Comment:
With c++20 char8_t is used for utf-8 characters. Would that not better here as well?

Reply:
Do we have this somewhere defined? I didn't know.

Reply:
this knowledge is provided by your always up-to-date online documentation system cppreference: https://en.cppreference.com/w/cpp/language/character_literal